src/mlpack/methods/ann/layer/one_hot_layer.hpp

```diff
@@ -58,7 +58,7 @@ class OneHotLayer
     output = inputActivations;
     output.zeros();
 
-    arma::uword maxIndex;
+    arma::uword maxIndex = 0;
```

Comment:
I looked through the armadillo code, `op_min::direct_min` equates `index_of_min_val` to zero by default.